### PR TITLE
Refract Examples

### PR DESCRIPTION
**Pull request by smizell**
These are some simple examples for the spec, seconds 1, 3, and most of 4 that is worth showing examples.

@zdne @fosrias 

Note: merging into refract branch here, not master.

### CODE REVIEW

**Comment by zdne** [comment]
Hey @smizell! I have just stumbled upon this – is it still valid PR? (read: should I review it and merge)

File: examples/Refract MSON Specification.md [hunk 1]
```diff
@@ -111,6 +111,42 @@ _Type_ ⇒ _[Validations][]_ _[opt]_
         - street
     ```
 
+    Refract:
+
+    ```js
+    {
+      'element': 'object',
```
Comment (kylef) [inline examples/Refract MSON Specification.md]: Perhaps we should use double quotes here so this is valid JSON?

**Comment by smizell** [comment]
We were putting this together in an attempt to explore using Refract for MSON. I think we can close as we have a namespace in place already. If you want to keep exploring, let me know, but also feel free to close.

**Comment by zdne** [comment]
OK. Closing. Reopen if someone wants to refresh it. Thanks :parking:

**Pull request closed without merging**